core/src/executor/execute.rs

```diff
@@ -259,8 +259,8 @@ pub async fn execute<T: GStore + GStoreMut>(
             });
 
             match rows {
-                RowsData::Insert(rows) => storage.insert_data(table_name, rows).await,
-                RowsData::Update(rows) => storage.update_data(table_name, rows).await,
+                RowsData::Insert(rows) => storage.append_data(table_name, rows).await,
+                RowsData::Update(rows) => storage.insert_data(table_name, rows).await,
```

Comment:
`RowsData` is Update, but the storage method it uses is `insert_data`. How about unifying this?

Reply:
Oh!! Thanks a lot, I missed it